master/getting-started/kubernetes/installation/migration-from-flannel.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Migrating a cluster from flannel to Calico
+---
```

Comment:
Recommend getting rid of jargon and simplifying...

# About

This article describes how migrate an existing Kubernetes cluster with Flannel networking to use Calico
networking. Calico provides a migration tool that performs a rolling update of the nodes in the cluster.
At the end, you will have a fully-functional Calico cluster using VXLAN networking between pods.

 # Prerequisites

- A cluster with Flannel for networking using the VXLAN backend.
- Flannel version v0.9.1 or greater (Canal version v3.7.0 or greater).
- Flannel must be configured to use the Kubernetes API for storing its configuration (as opposed to etcd).
- Flannel must have been installed using a Kubernetes daemonset.
- Cluster must allow for modification and removal of the Flannel daemonset. 
   For example, it must not be installed using the Kubernetes add-on manager.